geometric/engine.py

```diff
@@ -696,7 +696,7 @@ def calc_new(self, coords, dirname):
         self.schema_traj.append(ret)
 
         if ret["success"] is False:
-            raise ValueError("QCEngineAPI computation did not execute correctly.")
+            raise ValueError("QCEngineAPI computation did not execute correctly.\n" + ret["error_message"])
```

Comment:
This looks good to me. Travis should be passing, but allowed failures is not triggering correctly. Perhaps just comment out the `PROG=LATEST` sections in the `.travis.yml`.